lms/static/js/certificates/views/certificate_invalidation_view.js

```diff
@@ -35,11 +35,15 @@
                     var notes = this.$("#certificate-invalidation-notes").val();
                     var message = "";
 
-                    var certificate_invalidation = new CertificateInvalidationModel({
-                        url: this.collection.url,
-                        user: user,
-                        notes: notes
-                    });
+                    var certificate_invalidation = new CertificateInvalidationModel(
+                        {
+                            user: user,
+                            notes: notes
+                        },
+                        {
+                            url: this.collection.url
+                        }
+                    );
```

Comment:
Model's first parameter should only have model attributes, options should be passed as second parameter.